### PR TITLE
Modified some document specifications

### PR DESCRIPTION
**Pull request by justedennnnn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/setup/install/helm/index.md [hunk 1]
```diff
@@ -122,9 +121,9 @@ $ helm status istiod -n istio-system
     $ kubectl delete namespace istio-system
     {{< /text >}}
 
-## 卸载稳定的版本标签资源{#uninstall-stable-revision-label-resources}
+## 卸载稳定的版本标签资源 {#uninstall-stable-revision-label-resources}
 
-如果你决定继续使用旧的控制平面不更新，您可以通过第一次发布来卸载较新的版本及其标记 `helm template istiod istio/istiod -s templates/revision-tags.yaml --set revisionTags={prod-canary} --set revision=canary -n istio-system | kubectl delete -f -`。你必须按照上面的卸载程序卸载Istio 的修订版。
+如果你决定继续使用旧的控制平面不更新，您可以通过第一次发布来卸载较新的版本及其标记 `helm template istiod istio/istiod -s templates/revision-tags.yaml --set revisionTags={prod-canary} --set revision=canary -n istio-system | kubectl delete -f -`。你必须按照上面的卸载程序卸载 Istio 的修订版。
```
Comment (windsonsea) [inline content/zh/docs/setup/install/helm/index.md:126]:
```suggestion
如果您决定继续使用旧的控制平面不更新，您可以通过第一次发布来卸载较新的版本及其标记 `helm template istiod istio/istiod -s templates/revision-tags.yaml --set revisionTags={prod-canary} --set revision=canary -n istio-system | kubectl delete -f -`。您必须按照上面的卸载程序卸载 Istio 的修订版。
```
istio prefers to use 您

File: content/zh/docs/setup/install/helm/index.md [hunk 2]
```diff
@@ -8,8 +8,7 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
-请跟随本指南一起，使用
- [Helm](https://helm.sh/docs/) 安装、配置、并深入评估 Istio 网格系统。
+请跟随本指南一起，使用 [Helm](https://helm.sh/docs/) 安装、配置、并深入评估 Istio 网格系统。
```
Comment (windsonsea) [inline content/zh/docs/setup/install/helm/index.md:11]:
```suggestion
请遵循本指南的说明，使用 [Helm](https://helm.sh/docs/) 安装和配置 Istio 网格系统。
```
The en text is inconsistent with zh currently. Hope you can check this page through.

**Review by windsonsea — COMMENTED** [review]
The en text has changed much since the last translation to zh. Hope you can check it through, thanks.

**Comment by windsonsea** [comment]
Hi @justedennnnn, would you like to move this forward after a month passed?

If no any response in weeks, this will be closed. Thanks for your understanding.

**Comment by windsonsea** [comment]
This PR will be closed because:
- The author ignored all comments for a long time
- This PR has big problem of missing content to sync

**Pull request closed without merging**